### PR TITLE
no-AA-eviction-RoF.md

### PR DESCRIPTION
**Pull request by AlonMagrafta**
I forgot that

### CODE REVIEW

**Comment by AlonMagrafta** [comment]
@kaitlynmichael I forgot that, can you also verify my syntax of the notes, I have a feeling I might messed up with the spaces ;)

File: content/rs/administering/creating-databases/create-active-active.md [hunk 1]
```diff
@@ -114,6 +114,12 @@ Every instance of an Active-Active database can receive write operations, and al
     - [**Data eviction policy**]({{< relref "/rs/administering/database-operations/eviction-policy.md" >}}) -
         When the total size of the database reaches its memory limit,
         the database cannot accept new keys. You can select a data eviction policy that defines which keys to delete to make room for new keys. The default eviction policy for Active-Active is `noeviction`.
+        
+    {{< note >}}
+
```
Comment (kaitlynmichael) [inline content/rs/administering/creating-databases/create-active-active.md:119]:
```suggestion
```

File: content/rs/administering/creating-databases/create-active-active.md [hunk 2]
```diff
@@ -114,6 +114,12 @@ Every instance of an Active-Active database can receive write operations, and al
     - [**Data eviction policy**]({{< relref "/rs/administering/database-operations/eviction-policy.md" >}}) -
         When the total size of the database reaches its memory limit,
         the database cannot accept new keys. You can select a data eviction policy that defines which keys to delete to make room for new keys. The default eviction policy for Active-Active is `noeviction`.
+        
+    {{< note >}}
+    
+- Data eviction policy is not supported for Active-Active databases with Redis on Flash (RoF).
+
```
Comment (kaitlynmichael) [inline content/rs/administering/creating-databases/create-active-active.md:121]:
```suggestion
```

File: content/rs/administering/creating-databases/create-active-active.md [hunk 3]
```diff
@@ -114,6 +114,12 @@ Every instance of an Active-Active database can receive write operations, and al
     - [**Data eviction policy**]({{< relref "/rs/administering/database-operations/eviction-policy.md" >}}) -
         When the total size of the database reaches its memory limit,
         the database cannot accept new keys. You can select a data eviction policy that defines which keys to delete to make room for new keys. The default eviction policy for Active-Active is `noeviction`.
+        
+    {{< note >}}
+    
+- Data eviction policy is not supported for Active-Active databases with Redis on Flash (RoF).
```
Comment (kaitlynmichael) [inline content/rs/administering/creating-databases/create-active-active.md:120]:
```suggestion
- Data eviction policies are not supported for Active-Active databases with Redis on Flash (RoF).
```

File: content/rs/administering/database-operations/eviction-policy.md [hunk 4]
```diff
@@ -29,6 +29,12 @@ The eviction policy mechanism for Active-Active databases kicks in earlier than
 
 In case of network issues between Active-Active instances, memory can only be freed when all instances are in sync. If there is no communication between participating clusters, it can result in eviction of all keys and the instance reaching an Out-of-Memory state.
 
+   {{< note >}}
+
```
Comment (kaitlynmichael) [inline content/rs/administering/database-operations/eviction-policy.md:33]:
```suggestion
```

File: content/rs/administering/database-operations/eviction-policy.md [hunk 5]
```diff
@@ -29,6 +29,12 @@ The eviction policy mechanism for Active-Active databases kicks in earlier than
 
 In case of network issues between Active-Active instances, memory can only be freed when all instances are in sync. If there is no communication between participating clusters, it can result in eviction of all keys and the instance reaching an Out-of-Memory state.
 
+   {{< note >}}
+    
+- Data eviction policy is not supported for Active-Active databases with Redis on Flash (RoF).
+
```
Comment (kaitlynmichael) [inline content/rs/administering/database-operations/eviction-policy.md:35]:
```suggestion
```

File: content/rs/administering/database-operations/eviction-policy.md [hunk 6]
```diff
@@ -29,6 +29,12 @@ The eviction policy mechanism for Active-Active databases kicks in earlier than
 
 In case of network issues between Active-Active instances, memory can only be freed when all instances are in sync. If there is no communication between participating clusters, it can result in eviction of all keys and the instance reaching an Out-of-Memory state.
 
+   {{< note >}}
+    
+- Data eviction policy is not supported for Active-Active databases with Redis on Flash (RoF).
```
Comment (kaitlynmichael) [inline content/rs/administering/database-operations/eviction-policy.md:34]:
```suggestion
- Data eviction policies are not supported for Active-Active databases with Redis on Flash (RoF).
```

**Review by kaitlynmichael — CHANGES_REQUESTED** [review]

**Review by kaitlynmichael — APPROVED** [review]

**Pull request merged**